### PR TITLE
Remove Duplicated Code @WeightInitialization

### PR DESCRIPTION
**Pull request by jijoongmoon**
Remove Duplicated Code @WeightInitialization Fuc.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/src/layers.cpp [hunk 1]
```diff
@@ -182,65 +182,55 @@ float ReluPrime(float x) {
...
+          break;
+        case Layers::WEIGHT_XAVIER_UNIFORM:
+          f = RandUniform<float>(-1.0 * sqrt(6.0 / (height + width)), sqrt(6.0 / (height + width)));
+          break;
+        case Layers::WEIGHT_HE_UNIFORM:
+          f = RandUniform<float>(-1.0 * sqrt(6.0 / (height)), sqrt(6.0 / (height)));
+          break;
+        default:
+          f = 0;
+          break;
```
Comment (myungjoo) [inline nntrainer/src/layers.cpp:230]: Isn't this slower? or compilers will do the dirty work?

Comment (jijoongmoon) [reply nntrainer/src/layers.cpp:230]: Changed. Thanks.

File: nntrainer/src/layers.cpp [hunk 2]
```diff
@@ -182,65 +182,55 @@ float ReluPrime(float x) {
...
-          W.setValue(0, j, i, f);
-        }
-    } break;
-    default:
-      W.setZero();
-      break;
+  for (unsigned int i = 0; i < width; ++i) {
+    for (unsigned int j = 0; j < height; ++j) {
+      float f;
+      switch (init_type) {
```
Comment (dongju-chae) [inline nntrainer/src/layers.cpp:209]: Hmm.. I think the switch (or if) statement inside loop would be not good in the overall performance.
How about this style which adds common codes to the macro?
```
#define SOME_MACRO(x) for (...) for (...) ...

switch (init_type)
  case Layers::WEIGHT_LECUN_NORMAL:
     SOME_MARCO (RandNormal<float>(0, sqrt(1 / height)))
     break;
  ... 
```

Comment (jijoongmoon) [reply nntrainer/src/layers.cpp:209]: well. the code is not the newest. Thanks for the suggestion.

**Pull request merged**